yapftests/main_test.py

```diff
@@ -137,3 +137,11 @@ def testHelp(self):
       self.assertIn('indent_width=4', help_message)
       self.assertIn('The number of spaces required before a trailing comment.',
                     help_message)
+  def testExtraBlankLine(self):
```

Comment:
There needs to be a blank line above this. Also, I think the test is in the wrong place. It can go in one of the  `reformatter_*_tests.py` files.